scipy/stats/_continuous_distns.py

```diff
@@ -6488,14 +6494,13 @@ def _entropy(self):
         return _EULER/2.0 + 1 - 0.5*np.log(2)
 
     @extend_notes_in_docstring(rv_continuous, notes="""\
-        When the scale parameter is fixed by using the `fscale` argument,
-        this function uses the default optimization method to determine
-        the MLE. If the location parameter is fixed by using the `floc`
-        argument, the analytical formula for the estimate of the scale
-        is used. If neither parameter is fixed, the analytical MLE for
-        `scale` is used as a function of `loc` in numerical optimization
-        of `loc`, injecting corresponding analytical optimum for
-        `scale`.\n\n""")
+        Notes specifically for rayleigh.fit: If the location is fixed with
```

Comment:
Should these be showing up in the documentation for [`rayleigh`](https://21874-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.stats.rayleigh.html#scipy.stats.rayleigh)? I'm not seeing them.

Reply:
It should show up in the docstring of `rayleigh.fit`.   In IPython, I can see that text in the docstring when viewed with, e.g. `rayleigh.fit?`.